<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="3" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-04-09T00:40:44 IST" name="Suite" finished-at="2023-04-09T00:41:00 IST" duration-ms="16021">
    <groups>
    </groups>
    <test started-at="2023-04-09T00:40:44 IST" name="Test" finished-at="2023-04-09T00:41:00 IST" duration-ms="16021">
      <class name="TestNgTest.TestExample1">
        <test-method signature="StringCheck(java.lang.String)[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:44 IST" name="StringCheck" finished-at="2023-04-09T00:40:44 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Cannot inject @Test annotated Method [StringCheck] with [class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [StringCheck] with [class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:439)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:353)
at org.testng.internal.Parameters.createParameters(Parameters.java:708)
at org.testng.internal.Parameters.handleParameters(Parameters.java:884)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StringCheck -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:44 IST" name="beforeMethod" finished-at="2023-04-09T00:40:47 IST" duration-ms="3389" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="navigateToAUrl()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:47 IST" name="navigateToAUrl" finished-at="2023-04-09T00:40:49 IST" duration-ms="1985" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigateToAUrl -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:49 IST" name="afterMethod" finished-at="2023-04-09T00:40:49 IST" duration-ms="200" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:49 IST" name="beforeMethod" finished-at="2023-04-09T00:40:50 IST" duration-ms="914" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="navigateToAUrl1()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:50 IST" name="navigateToAUrl1" finished-at="2023-04-09T00:40:52 IST" duration-ms="1939" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigateToAUrl1 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:52 IST" name="afterMethod" finished-at="2023-04-09T00:40:52 IST" duration-ms="146" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:52 IST" name="beforeMethod" finished-at="2023-04-09T00:40:53 IST" duration-ms="787" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="navigateToAUrl2()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:53 IST" name="navigateToAUrl2" finished-at="2023-04-09T00:40:55 IST" duration-ms="1279" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<Facebook – log in or sign up> but was:<Facebook-log in or sign up>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Facebook – log in or sign up> but was:<Facebook-log in or sign up>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
at TestNgTest.TestExample1.navigateToAUrl2(TestExample1.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigateToAUrl2 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:55 IST" name="afterMethod" finished-at="2023-04-09T00:40:55 IST" duration-ms="128" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:55 IST" name="beforeMethod" finished-at="2023-04-09T00:40:55 IST" duration-ms="842" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="navigateToAUrl3()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:40:56 IST" name="navigateToAUrl3" finished-at="2023-04-09T00:41:00 IST" duration-ms="4052" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigateToAUrl3 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestNgTest.TestExample1@3f6b0be5]" started-at="2023-04-09T00:41:00 IST" name="afterMethod" finished-at="2023-04-09T00:41:00 IST" duration-ms="199" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- TestNgTest.TestExample1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
